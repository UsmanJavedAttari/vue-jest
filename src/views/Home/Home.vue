<template>
    <div>This is home!</div>
</template>

<script lang="ts" src="./Home.ts" />
