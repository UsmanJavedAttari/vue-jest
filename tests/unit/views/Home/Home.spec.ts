import Home from '@/views/Home/Home';
import { mount, Wrapper } from '@vue/test-utils';

describe('Home Component', () => {
    let wrapper: Wrapper<Home>;

    beforeEach(() => {
        wrapper = mount(Home);
    });

    test('Verify Snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
