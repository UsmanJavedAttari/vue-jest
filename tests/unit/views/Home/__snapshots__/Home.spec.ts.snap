// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Verify Snapshot 1`] = `<div>This is home!</div>`;
